pandas/io/excel.py

```diff
@@ -137,7 +137,7 @@
 na_values : scalar, str, list-like, or dict, default None
     Additional strings to recognize as NA/NaN. If dict passed, specific
     per-column NA values. By default the following values are interpreted
-    as NaN: '""" + fill("', '".join(sorted(_NA_VALUES)), 70) + """'.
+    as NaN: '""" + fill("', '".join(sorted(_NA_VALUES)), 999) + """'.
```

Comment:
why this change?

Reply:
Ah, I see you mentioned this in the very first post of the PR. But, I don't think this is the fully correct solution, as this will make the line too long. 
Apparently for the `read_csv` docstring this is working properly, so maybe check how they did it there?

Reply:
I think that the long line is not a problem, as line breaks in the input doc string do not translate to line breaks in the compiled output. I have tested compiling the docs locally and the output looked fine for me. However, since you referred to the read_csv doc string, I had a look at that. It uses `, subsequent_indent="    "`, so I will change it here to keep things consistent.